src/Raven.Server/Web/Studio/FolderPath.cs

```diff
@@ -110,6 +136,11 @@ private static string GetRestrictedFolder(bool isBackupFolder, RavenConfiguratio
 
     public class FolderPathOptions
     {
-        public List<string> List { get; } = new List<string>();
+        public List<string> List { get; private set; } = new List<string>();
+
+        public void Sort()
+        {
+            List = List.OrderBy( O => O ).ToList();
```

Comment:
Please follow formatting rules. Also it should be `(o => o)`

Reply:
Removed